app/client/src/features/events/EventLive.tsx

```diff
@@ -55,7 +61,10 @@ export const EVENT_LIVE_QUERY = graphql`
             id
             ... on Event {
                 isViewerModerator
+                startDateTime
+                isActive
```

Comment:
Can just get these under the `EVENT_DETAILS_CARD_FRAGMENT`

Reply:
`EVENT_DETAILS_CARD_FRAGMENT` only has `startDateTime` along with 4 other fields that aren't needed. Using the fragment for just 1 field that would also bring in extraneous fields that we don't need seems unnecessary in this case?